tools/cldr-apps/js/src/esm/cldrLoad.js

```diff
@@ -780,17 +780,26 @@ function loadAllRows(itemLoadInfo, theDiv) {
   );
   const url = cldrTable.getPageUrl(curLocale, curPage, curId);
   $("#nav-page").show(); // make top "Prev/Next" buttons visible while loading, cf. '#nav-page-footer' below
-  myLoad(url, "page", function (json) {
-    loadAllRowsFromJson(json, theDiv);
-  });
+  cldrAjax
+    .doFetch(url)
+    .then((response) => response.json())
+    .then((json) => loadAllRowsFromJson(json, theDiv))
+    .catch((err) => {
```

Comment:
this will catch any error, even if the `loadAllRowsFromJson()` function throws.

Reply:
excellent modernization!